cmd/tailscale/cli/serve_v2_test.go

```diff
@@ -775,6 +776,15 @@ func TestServeDevConfigMutations(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "no_http_with_funnel",
+			steps: []step{
+				{
+					command: cmd("funnel --http=80 3000"),
+					wantErr: exactErrMsg(errors.New("error parsing commandline arguments: flag provided but not defined: -http")),
```

Comment:
looks good, but if the underlying library changes anything in this message, it will fail.

Reply:
👍 changed to anyError